package.json

```diff
@@ -22,6 +22,7 @@
     "md5": "2.2.1",
     "node-cache": "4.2.0",
     "request": "2.88.0",
+    "request-promise-native": "^1.0.8",
```

Comment:
This looks like it's been deprecated. There are some alternatives listed.

https://github.com/request/request-promise-native#deprecated

Reply:
Sorry - it's request that has been deprecated. And we're already using that, not adding it in this PR.

Reply:
yeah I had the same thoughts, but unless we want to move off `request` we have to stay deprecated :(